### DISCUSSION TITLE
Question about performance

### DISCUSSION THREAD

**Discussion by ekerstein** [chartjs/chartjs-plugin-deferred#18] [Q&A]
Does this library help the performance of a page, or is it more about deferring animation for better user experience?

I have a page with lots of charts on it and I'm wondering if it would load faster if I use this plugin to defer the initial chart updates until the user scrolls down.

**Comment by BR0kEN-** [comment]
> Does this library help the performance of a page?

Depends on what you call performance.

> is it more about deferring animation for better user experience?

It is definitely for that.

> would it load faster if I use this plugin to defer the initial chart updates until the user scrolls down?

It would not load faster but decrease a screen freeze seen when multiple charts render at once. Treat it as lazy loading.
